soh/soh/Enhancements/mods.cpp

```diff
@@ -258,6 +256,52 @@ void RegisterRupeeDash() {
     });
 }
 
+void RegisterBonkDamage() {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnPlayerBonk>([]() {
+        uint8_t bonkOption = CVarGetInteger("gBonkDamageMul", 0);
+        uint16_t bonkDamage = 0;
+        switch (bonkOption) {
+            // Quarter heart
+            case 1:
```

Comment:
If you want to do something fancy, this whole switch statement could be done in just two expressions:

```
switch (bonkOption) {
    case 1:
    case 2:
    case 3:
    case 4:
    case 5:
    case 6:
        // 4 damage = quarter heart
        bonkDamage = 1 << (1 + bonkOption);
        break;
    default:
        // Also covers cases 0 and 7
        break;
}
```

Reply:
i like the flexibility of the current system, if we want to add a non `2^x` value we can do that easily by just adding another case